dotCMS/src/main/java/com/dotcms/util/MimeTypeUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.dotcms.util;
+
+import com.dotcms.tika.TikaUtils;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.rainerhahnekamp.sneakythrow.Sneaky;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Mime Type Utils
+ * @author jsanca
+ */
+public class MimeTypeUtils {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4772115)